addons/l10n_fi/tests/test_references.py

```diff
@@ -0,0 +1,70 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import unittest
```

Comment:
I think this file is an exaggeration.  Previous file already tests more than we do for the other localizations.

Reply:
Well it doesn't harm and it could save us from unintended changes. I'd keep it.

Reply:
The try..except is not too catholic then. We might use assertRaises or something instead.  It seems like easier to integrate it in the previous tests.  But if you don't mind adapting this one, we can keep it.